.travis.yml

```diff
@@ -1,20 +1,28 @@
 language: python
```

Comment:
It may be better to transform this into a GitHub Actions YAML file now that Travis does not provide free execution time for open source projects anymore :(